independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/handlers/ClientSendRequestHandler.java

```diff
@@ -165,6 +167,12 @@ public void handle(AsyncResult<AsyncFile> openedAsyncFile) {
                                             attachSentHandlers(sent, httpClientRequest, requestContext);
                                         }
                                     });
+                } else if (requestContext.isInputStreamUpload()) {
+                    Future<HttpClientResponse> sent = httpClientRequest.send(
+                            new InputStreamReadStream(
+                                    Vertx.currentContext().owner(), (InputStream) requestContext.getEntity().getEntity(),
```

Comment:
Not sure - I believe you want the connection one.

Reply:
We have already been using the same thing in Multipart handling without issue

Reply:
Ok.